### ISSUE TITLE
'nwr' not working but 'way' does.

### ISSUE THREAD

**Issue by DaveF63** [drolbr/Overpass-API#494]
Hi

This works as expected, returning a number of ways
```
[bbox:{{bbox}}];
nwr[railway=station];
foreach->.Elem(
 nwr(around.Elem:150)[building];
  way._[building=retail];
  make CntLst
    nodes = count(nodes),
    ways = count(ways),
    rels = count(relations);
     out;  
);
```
http://overpass-turbo.eu/s/Asw

But after changing `way` to `nwr` this doesn't (it does return a relation which the above doesn't do)
```
[bbox:{{bbox}}];
nwr[railway=station];
foreach->.Elem(
 nwr(around.Elem:150)[building];
  nwr._[building=retail];
  make CntLst
    nodes = count(nodes),
    ways = count(ways),
    rels = count(relations);
     out;  
);
```
http://overpass-turbo.eu/s/Asx

Why is this?

**Comment by mmd-osm** [comment]
Thanks for reporting, that's a bug. To demonstrate the issue you can simplify the second example to:

[bbox:{{bbox}}];
```
nwr[building];
nwr._[building=retail];
out geom; 
```

**Comment by drolbr** [comment]
Again thank you for reporting the problem. Fixed in 8e0f78b1cc2c4510b9706dc11a505573c6f6d773
